LLAP/llap_restart.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
For copying this file into the cluster master, you can do something similiar to this: https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/mlvm/mlvm.sh#L26

And update the readme telling the user to provide their own gcs url if they are maintaining their own copy of the restart script, like here: https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/mlvm/README.md#using-this-initialization-action

You will also need to update the readme with the location within master node where the restart script will be kept.

Reply:
done